speech/speech_samples.rb

```diff
@@ -88,24 +131,36 @@ def get_async_operation_results operation_name:
   # [END get_async_operation_results]
 end
 
+require "google/cloud/speech"
+
 if __FILE__ == $PROGRAM_NAME
-  command = ARGV.shift
+  project_id = Google::Cloud::Speech.new.project_id
```

Comment:
We do this in a number of places: [datasets.rb](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/6aa58d6ec26c306a75e0b5257b17ad68edc3b4f9/bigquery/datasets.rb#L69), [tables.rb](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/6aa58d6ec26c306a75e0b5257b17ad68edc3b4f9/bigquery/tables.rb#L227), [language_samples.rb](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/b33f4f395a35c7a8f5676ff24b61b3673f834635/language/language_samples.rb#L120)